manifests/README.md

```diff
@@ -0,0 +1,40 @@
+# TL;DR
+If you want to skip any customization, you can deploy Kubeflow Pipelines by running
+```
+export PIPELINE_VERSION=0.1.20
+kubectl create -f https://raw.githubusercontent.com/kubeflow/pipelines/$PIPELINE_VERSION/manifests/namespaced-install.yaml
+```
+
+You might lack the permission to create role and command might partially fail. If so, bind your account as cluster admin.
+(Or role creator in your namespace)
+```
+kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=[your-user-name]
+```
+
+# Customization
```

Comment:
Is there any other customization that user want to perform other than namespace?
If there is, I suggest to add instructions as well.

Reply:
sounds good to me. Feel free to ping me when you add more customization.